r/tests/testthat/test-atac_utils.R

```diff
@@ -248,6 +248,11 @@ test_that("write_insertion_bed works", {
 })
 
 test_that("macs_e2e_works", {
+  # only run if macs2 is installed
+  if ((suppressWarnings((system2("macs2", args = "--version", stdout = FALSE, stderr = FALSE) != 0)) ||
+      (!grepl("macs", system2("macs2", args = "--version", stdout = TRUE))))) {
+    skip("macs2 not installed")
+  }
```

Comment:
Two requested changes in this test:
1. Can we make it work with either macs2 or macs3 and just skip if neither is available?
2. Could we make the `cell_groups` in this test be `v o w e l` and `consonant;` to help check that our quoting is working properly?